RadarSDK/RadarAPIClient.m

```diff
@@ -344,7 +344,9 @@ - (void)updateTripWithOptions:(RadarTripOptions *)options status:(RadarTripStatu
         params[@"destinationGeofenceExternalId"] = options.destinationGeofenceExternalId;
     }
     params[@"mode"] = [Radar stringForMode:options.mode];
-
+    if (options.scheduledArrivalAt) {
+        params[@"scheduledArrivalAt"] = [[RadarUtils isoDateFormatter] stringFromDate:options.scheduledArrivalAt];
```

Comment:
A quick question about this line:
- Is there any difference between `[RadarUtils isoDateFormatter]` vs `RadarUtils.isoDateFormatter` (found [here](https://github.com/radarlabs/radar-sdk-ios/blob/ef6cc9870313cef842ec04bcbaf409c5b4d3fe6c/RadarSDK/RadarLog.m#L49) and [here](https://github.com/radarlabs/radar-sdk-ios/blob/6a68647a56b82a56a6c6c9d7839d3d57b937ff05/RadarSDK/RadarEvent.m#L117))?
- Does `isoDateFormatter` take in both `string` arguments and `date` arguments? (This may be a faulty assumption, but we're calling `stringFromDate` here and in RadarLog.m, but calling `dateFromString` in RadarEvent.m)

Reply:
* In Objective-C, calling a function (`[RadarUtils isoDateFormatter]`) that takes no arguments is identical to accessing it like a property (`RadarUtils.isoDateFormatter`), so there's no difference at all, other than style. (I prefer `RadarUtils.isoDateFormatter`; I think that @nickpatrick prefers `[RadarUtils isoDateFormatter]`.)

* Yes, all of the various subclasses of [`NSFormatter`](https://developer.apple.com/documentation/foundation/nsformatter) have `stringFromX()` and `xFromString()` functions for converting between them.